worker/worker-execution/score-worker-execution-impl/src/main/java/io/cloudslang/worker/execution/services/ExecutionServiceImpl.java

```diff
@@ -547,22 +547,48 @@ private static boolean useDefaultGroup(Execution execution) {
         return useDefaultGroup;
     }
 
-    public static void postExecutionSettings(Execution execution) {
-        // Decide on Group
+    public void postExecutionSettings(Execution execution) {
```

Comment:
final method param

Reply:
Let's say that I do create a new instance of type Execution in my method now that I do not have final on the parameter. That new instance would be completely disregarded. That would be a very bad programming on my behalf and that code should be rejected on the spot.
I don't see any value in having the final keyword used, this is not even a best practice on CS, I would have to change the other methods as well to be consistent